yast_rdp.pm

```diff
@@ -0,0 +1,90 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: this test checks that YaST2's RDP module is behaving
+#          correctly and verifying that RDP service has been
+#          successfully set.
+# Maintainer: Jun Wang <jgwang@suse.com>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use version_utils 'is_sle';
+
+my $repos = {'12-SP1' => 'http://download.suse.de/ibs/Devel:/YaST:/SLE-12-SP1/SLE-12-SP1/',
+    '12-SP1-UP' => 'http://download.suse.de/ibs/Devel:/YaST:/SLE-12-SP1/SLE-12-SP1_Update/',
+    '12-SP2'    => 'http://download.suse.de/ibs/Devel:/YaST:/SLE-12-SP2/SLE_12_SP2/',
+    '12-SP2-UP' => 'http://download.suse.de/ibs/Devel:/YaST:/SLE-12-SP2/SUSE_SLE-12-SP2_Update/',
+    '12-SP3'    => 'http://download.suse.de/ibs/Devel:/YaST:/SLE-12-SP3/SLE_12_SP3/',
+    '12-SP3-UP' => '',
+    '12-SP4'    => 'http://download.suse.de/ibs/Devel:/YaST:/SLE-12-SP4/SLE_12_SP4/',
+    '12-SP4-UP' => 'http://download.suse.de/ibs/Devel:/YaST:/SLE-12-SP4/SUSE_SLE-12-SP4_Update/'
```

Comment:
I am kinda curious which product is being tested, or to rephrease my question, is tested scenario something that is supported?

Reply:
I just had a try to support more products. If we only test yast2-rdp in the default repos, I can adjust the code.